web-animations-1/Overview.bs

```diff
@@ -957,7 +988,9 @@ an animation, <var>animation</var>, to <var>seek time</var> is as follows:
             <a lt="inactive timeline">inactive</a>, or
         *   <var>animation</var>'s [=playback rate=] is 0,
 
-    ::  Set <var>animation</var>'s <a>hold time</a> to <var>seek time</var>.
+    ::  Set <var>animation</var>'s <a>hold time</a> to <var>seek time</var>
+    ::  Set <var>animation</var>'s <a>hold phase</a> to
+        [=timeline active phase=].
```

Comment:
I'm probably do /[=timeline active phase=]/[=timeline active phase|active=]/ (or whatever the syntax is for making it just say "active"). I think you did that later in the PR so we should probably do the same everywhere.